### PR TITLE
ci: add GHA to build maru

### PR DESCRIPTION
**Pull request by FurqanHabibi**
Will build for both x64 and arm and push to HENNGE's ghcr.io registry.

### CODE REVIEW

**Review by jarviliam — APPROVED** [review]
zzzz

File: .github/workflows/build_maru.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v5
+        with:
+          push: true  
+          tags: ghcr.io/HENNGE/maru:latest
```
Comment (hanscg) [inline .github/workflows/build_maru.yaml:48]: You don't need to create a repository to push this?

Comment (FurqanHabibi) [reply .github/workflows/build_maru.yaml:48]: Apparently not, just like dynalite which is different than it's repo name dynalite-docker
https://github.com/HENNGE/dynalite-docker/pkgs/container/dynalite

**Review by hanscg — APPROVED** [review]
LGTM 👍

**Pull request merged**